notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6328 @@
+{
```

Comment:
<p>"...building a Retrieval-Augmented Generation (RAG)..."</p><p> </p><p>"...in the <a href="#choose-data-and-query-vectorisation-options" target="_blank">Choose data and query vectorisation options</a> section below!"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>👍</p>